migration-engine/connectors/sql-migration-connector/src/flavour/postgres.rs

```diff
@@ -11,11 +12,18 @@ use user_facing_errors::{
 };
 
 #[derive(Debug)]
-pub(crate) struct PostgresFlavour(pub(crate) PostgresUrl);
+pub(crate) struct PostgresFlavour {
+    pub(crate) url: PostgresUrl,
+    features: BitFlags<MigrationFeature>,
```

Comment:
We talked about it yesterday, but I'm not sure why this shows up in the diff. Let's see if it's still there after a rebase.